content/en/docs/examples/vm-bookinfo/index.md

```diff
@@ -32,7 +34,7 @@ https://docs.google.com/drawings/d/1G1592HlOVgtbsIqxJnmMzvy6ejIdhajCosxF1LbvspI/
 
 * Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application (in the `bookinfo` namespace).
 
-* Create a VM named 'vm-1' in the same project as Istio cluster, and [Join the Mesh](/docs/examples/mesh-expansion/single-network/).
+* Create a VM named 'vm-1' in the same project as Istio cluster, and [Join the Mesh]/docs/tasks/virtual-machines/single-network/).
```

Comment:
Is this example meant to be in a GCP-specific environment? "project" is a GCP word, not a generic word.

Reply:
This example, as written, is GCP-specific. For that reason, I don't think it's appropriate to elevate it to a top level example like this. Unless it can be rewritten more generically, and tested, I think it should probably be moved to a blog.

Reply:
@frankbu I'm going through the text to remove any references to GCP. Would that adress your concern?

Reply:
Could we use "environment" as the genetic term here instead of project?

Reply:
@rcaballeromx It's not quite that simple. This example is currently assuming that the VM is created using [these instructions](https://preliminary.istio.io/docs/examples/mesh-expansion/single-network/) which are also Google specific. Before I think it would be appropriate to make this a top level example, I think it needs to be changed to add options to do it with either network pattern, [single network](https://preliminary.istio.io/docs/examples/mesh-expansion/single-network/) or [multi-network](https://preliminary.istio.io/docs/examples/mesh-expansion/multi-network/). It would also need to be tested both ways to make sure it works.

Btw, I also think that the [single network](https://preliminary.istio.io/docs/examples/mesh-expansion/single-network/) instructions should be made more general so as not to specifically use Google commands.

It seems to me that there is more work to do with these mesh-expansion examples than just move them and tweak some wording. I think a similar problem exists with the [multicluster bookinfo example](https://preliminary.istio.io/docs/examples/multicluster/gke/#deploy-bookinfo-example-across-clusters). It should also be redone to work in a single or multi-network environment. I think we need to start opening issues, and hopefully get some people to help.

Reply:
@frankbu I agree with you that the content needs a lot of work to not only make it more platform agnostic, but also to update it to use some of the recently added features.
That said, this is just a first step with the following goals:

    - Provide clarity to users that are not familiar with the term "mesh expansion" and who are confused that there appears to be no content in istio.io related to VMs.
    - Ensure that all the VM related content can be found in a single place.
I'm updating the PR based on your feedback, but keep in mind that this is only the first step in a marathon.
We will need to iterate on this content moving forwards based on the feedback we get from the community and our users too.